### ISSUE TITLE
Is there any way to hide the Pin to Notification icon

### ISSUE THREAD

**Issue by Jjquigley** [mpcjanssen/simpletask-android#1206]
![Screenshot_20221006-133610](https://user-images.githubusercontent.com/18133162/194415149-e9902c61-173a-4e16-b00d-213aa4c6431d.png)
Using SimpleTask Dropbox version 11.0.1 on a Samsung Galaxy Luna Pro, Android version 6.0.1
On my phone, when I select several items and would like to change the threshold date, the new pin icon now crowds the threshold icon a bit too far to the right and I can no longer touch it. I wondered if there was a way to hide either the pin icon or the completion icon.  My workaround is to turn on screen rotation and turn the phone to landscape orientation. I assume there is no easy way to slightly shrink the icons.

**Comment by sphenotux** [comment]
Same behaviour here, v11.0 Nextcloud Edition.
(Did not make use of the threshold date as of now - just to confirm your description. Workaround screen rotation works as well.)
Huawei P40 lite.
Screen/display size matters?

**Comment by smichel17** [comment]
Yes, it's an issue only on smaller devices. Tentatively I'd say this is a duplicate of https://github.com/mpcjanssen/simpletask-android/issues/1182
